Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+#!groovy
+
+@Library('katsdpjenkins') _
+katsdp.setDependencies(['ska-sa/katsdpdockerbase/master'])
```

Comment:
Add 'ska-sa/katpoint/master' as well, so that this gets retested after katpoint is updated.